vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -309,9 +321,10 @@ def _apply_rotary_emb_neuron(
         key = key.view(num_tokens, -1, self.head_size)
 
         if self.rotary_dim == self.head_size:
-            query = _apply_rotary_emb(query, cos, sin, self.is_neox_style)
+            query = _apply_rotary_emb_torch(query, cos, sin,
+                                            self.is_neox_style)
```

Comment:
```suggestion
            query = _apply_rotary_emb(query, cos, sin, self.is_neox_style)
```

This change is unnecessary for neuron now.